src/main/kotlin/com/github/huntersherms/pojo2parquet/Pojo2Parquet.kt

```diff
@@ -0,0 +1,154 @@
+package com.github.huntersherms.pojo2parquet
+
+import org.apache.avro.reflect.ReflectData
+import org.apache.hadoop.fs.Path
+import org.apache.parquet.avro.AvroParquetReader
+import org.apache.parquet.avro.AvroParquetWriter
+import org.apache.parquet.hadoop.ParquetFileWriter
+import java.io.File
+import org.apache.parquet.hadoop.metadata.CompressionCodecName.GZIP
+import com.fasterxml.jackson.dataformat.avro.schema.AvroSchemaGenerator
+import com.fasterxml.jackson.dataformat.avro.AvroFactory
+import com.fasterxml.jackson.dataformat.avro.AvroGenerator
+import com.fasterxml.jackson.dataformat.avro.AvroMapper
+import com.fasterxml.jackson.dataformat.avro.AvroSchema
+import org.apache.avro.Schema
+import org.apache.avro.file.DataFileReader
+import org.apache.avro.file.SeekableByteArrayInput
+import org.apache.avro.generic.GenericData
+import org.apache.avro.generic.GenericDatumReader
+import org.apache.avro.generic.GenericDatumWriter
+import org.apache.avro.generic.GenericRecord
+import org.apache.avro.io.BinaryEncoder
+import org.apache.avro.io.EncoderFactory
+import java.io.ByteArrayOutputStream
+
+
+class Pojo2Parquet<T>(private val clazz: Class<T>) {
+
+    /**
+     * Converts a list of Pojos to a temp parquet file and returns it
+     *
+     * Uses reflection so that providing an Avro schema is not necessary
```

Comment:
```“So that” tells us for what purpose something is done.  What was the intention or plan?  In contrast, “so” tells us the effect.  It tells us what happened as a result.```